README.md

```diff
@@ -0,0 +1,17 @@
+# OpenRPC Specification for the Filecoin Common Node API
+
+This is an accessory [this FIP](TODO(aatifsyed): add link)
```

Comment:
@ansermino I couldn't find your fork :)

Reply:
Lets leave until we can link to the actual FIP